debugtools/DDR_VM/src/com/ibm/j9ddr/CompatibilityConstants29.dat

```diff
@@ -68,6 +68,8 @@ J9JavaClassFlags.J9ClassLargestAlignmentConstraintDouble = 0
 J9JavaClassFlags.J9ClassLargestAlignmentConstraintReference = 0
 J9JavaClassFlags.J9ClassRequiresPrePadding = 0
 
+J9JavaAccessFlags.J9StaticFieldIsNullRestricted = 0
```

Comment:
Please move this into the existing group for `J9JavaAccessFlags`.

Reply:
updated